src/app/screens/services/EmploiDuTemps.tsx

```
@@ -45,79 +203,151 @@
+          {Array.from({ length: TOTAL_HOURS }).map((_, index) => {
+            const hour = START_HOUR + index;
+            return (
+              <View key={hour} className="justify-start items-end pr-2">
+                {/* heure */}
+                <View style={{ height: HOUR_HEIGHT / 4 }} >
+                  <Text style={{ color: theme.text }}>{hour}h</Text>
+                </View>
+
+                {/* tirets (quart d’heure) */}
+                {Array.from({ length: 3 }).map((_, i) => (
+                  <View key={i} style={{ height: HOUR_HEIGHT / 4 }}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)